VoodooI2CHID/VoodooI2CMultitouchHIDEventDriver.cpp

```diff
@@ -382,6 +387,9 @@ void VoodooI2CMultitouchHIDEventDriver::handleDigitizerTransducerReport(VoodooI2
     }
 
     if (!has_confidence)
+        transducer->confidence.update(1, timestamp);
```

Comment:
Todo: calculate width/height and see if either of them go above 25mm if there isn't a confidence bit.
If there isn't any width/height fields...then I guess too bad?